pkg/limatype/lima_yaml.go

```diff
@@ -245,6 +246,10 @@ type Provision struct {
 	Playbook                        string             `yaml:"playbook,omitempty" json:"playbook,omitempty"` // DEPRECATED
 	// All ProvisionData fields must be nil unless Mode is ProvisionModeData
 	ProvisionData `yaml:",inline"` // Flatten fields for "strict" YAML mode
+	// ProvisionModeYQ borrows Path and Permissions from ProvisionData
+	Expression *string `yaml:"expression,omitempty" json:"expression,omitempty" jsonschema:"nullable"`
+	User       *string `yaml:"user,omitempty" json:"user,omitempty" jsonschema:"nullable"`
+	Format     *string `yaml:"format,omitempty" json:"format,omitempty" jsonschema:"nullable"`
```

Comment:
As I mentioned above, I think it should borrow "Owner" as well.

I'm undecided if `expression` should be `expr` for brevity. I already disliked `permissions` as too long, but `mode` was already taken.

Either `expr` or `expression` is fine with me, but would like to hear what others think!